docs/content/usage/configuration/credentials.md

```diff
@@ -5,6 +5,27 @@ draft: false
 description: "Configure credentials that the Infection Monkey will use for propagation."
 ---
 
-On this screen you can feed the Infection Monkey "stolen" credentials from your network, simulating an attacker with inside knowledge.
+On this screen, you can configure credentials for Infection Monkey to use during propagation.
+These can be anything from default or weak passwords, to "stolen" credentials from your
+network, simulating an attacker with inside knowledge.
 
 ![Configure credentials](/images/island/configuration_page/credentials_configuration.png "Configure credentials")
+
+The "Identity" field accepts usernames and email addresses. As seen in the screenshot, the
+supported secret types are passwords, LM hashes, NTLM hashes, and SSH keys.
```

Comment:
We should rename "Identity" column to "Username/Email" and delete this section. But probably in a different PR.

Reply:
Leaving it as is for now